tests/ui/components/results/directanswercomponent.js

```diff
@@ -0,0 +1,52 @@
+import { mount } from 'enzyme';
+import mockManager from '../../../setup/managermocker';
+import DOM from '../../../../src/ui/dom/dom';
+import StorageKeys from '../../../../src/core/storage/storagekeys';
+import DirectAnswerComponent from '../../../../src/ui/components/results/directanswercomponent';
+
+DOM.setup(document, new DOMParser());
+let COMPONENT_MANAGER, defaultConfig, directAnswerData, storage;
+beforeEach(() => {
+  COMPONENT_MANAGER = mockManager({
+    filterRegistry: {
+      getAllFilterNodes: () => []
+    }
+  });
+  storage = COMPONENT_MANAGER.core.storage;
+  const bodyEl = DOM.query('body');
+  DOM.empty(bodyEl);
+  DOM.append(bodyEl, DOM.createEl('div', { id: 'test-component' }));
+
+  defaultConfig = {
+    container: '#test-component',
+    verticalKey: 'people'
+  };
+
+  directAnswerData = {
+    answer: {
+      entityName: 'Amani Farooque',
+      fieldApiName: 'mainPhone',
+      fieldName: 'Phone Number',
+      fieldType: 'phone',
+      value: '+18003332222'
+    },
+    relatedItem: {
+      verticalConfigId: 'people',
+      data: {
+        id: 'Employee-2116',
+        type: 'ce_person',
+        website: 'http://www.test.com'
+      }
+    },
+    type: 'FIELD_VALUE'
+  };
+});
+
+describe('DirectAnswer component', () => {
+  it('displays the answer correctly', () => {
```

Comment:
Can we add a TODO to make this test more thorough? I think what you have is fine for now (especially since there were no tests to start with). Ultimately, we want to verify specific things about the display, such as the breadcrumb view at the top, that the displayed value is correct, etc.

Reply:
sure I can make an item!